types/api/device.d.ts

```diff
@@ -51,10 +51,25 @@ export interface ApiDeviceLocationFixup {
   location?: LatLng; // Supply a location to map to the station
 }
 
+type SettingsBase = {
+  updated: IsoFormattedDateString;
+};
+
+export type ThermalRecordingSettings = {
+  useLowPowerMode: boolean;
+} & SettingsBase;
+
+export type WindowsSettings = {
+  startRecording: string;
+  stopRecording: string;
+  powerOn: string;
+  powerOff: string;
+} & SettingsBase;
+
 export interface ApiDeviceHistorySettings {
+  location?: LatLng;
```

Comment:
I assume this would only be different from the `DeviceHistory.location` field in the `synced === false` case?